ansible/playbooks/roles/kubernetes_master/templates/calico.yml.j2

```diff
@@ -3861,6 +3861,10 @@ spec:
             # Set Felix endpoint to host default action to ACCEPT.
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
               value: "ACCEPT"
+{% if 'arm64' in calico_arch %}
+            - name: FELIX_IPTABLESBACKEND
+              value: "Legacy"
```

Comment:
Have you tried setting to "NFT" or "Auto"?
We use Calico 3.15 and according to docs "Legacy" is the default:
https://projectcalico.docs.tigera.io/archive/v3.15/reference/felix/configuration
but from 3.18 the default is "Auto":
https://projectcalico.docs.tigera.io/archive/v3.18/reference/felix/configuration

We install `iptables-ebtables` in `common` role and iptables-1.8.4 is installed as dependency. Here they suggest to set the backend to "NFT" for RHEL 8: https://github.com/kubernetes-sigs/kubespray/issues/9005

Reply:
1. Yes, i tried different options - didn't work as expected.
2. The question is if we should change (downgrade) calico version in task where we switch architecture version?

Reply:
We have only one (common) template of Calico config file, which may differ between Calico versions, that's why I would like to use the same Calico version for both architectures. It can be newer but then we would need to upgrade it for x86_64 as well to have the same.